### ISSUE TITLE
pymssql does not unserialize DATE column

### ISSUE THREAD

**Issue by pembo13** [pymssql/pymssql#409]
I have referenced http://pymssql.org/en/latest/faq.html#pymssql-does-not-unserialize-date-and-time-columns-to-datetime-date-and-datetime-time-instances

I have:
* pymssql 2.1.2
* SQL Server 10.50.4000 (2008 R2 Service Pack 2 )
* Centos 7 with Python 2.7
* freetds 0.95 (freetds-0.95.81-1.el7.x86_6, freetds-devel-0.95.81-1.el7.x86_64)3
* my freetds.conf contains `[global] tds version = 7.3`

Which seems to meet all the requirements, yet my DATE columns are coming back are unicode.

**Comment by rsyring** [comment]
Can you provide a small script as a test case showing the problem?

**Comment by pembo13** [comment]
```python
conn = connection(as_dict=False) ; cur = conn.cursor()
cur.execute("""SELECT GETDATE() date1, CAST(GETDATE() AS DATE) date2;""", (profileid,fromdate,todate))
```

which outputs:

```
[(datetime.datetime(2016, 3, 6, 19, 37, 51, 267000), u'2016-03-06')]
```

`connection` is just a function that creates a connection to the database with my credentials.

**Comment by ramiro** [comment]
Unserialization of such data type isn't available in 2.1.2 but on the latest in-development code. See http://pymssql.org/en/latest/changelog.html

Note the documentation you linked is the one that'll be included with the next feature release of pymssql (2.2.0). Compare that docs section with its 2.1.2 counterpart: http://pymssql.org/en/v2.1.2/faq.html#pymssql-does-not-unserialize-date-and-time-columns-to-datetime-date-and-datetime-time-instances 

You'll need to compile pymssql by yourself. from the master branch. 

Be aware it might be buggy/unstable and we welcome any feedback.

**Comment by pembo13** [comment]
Sorry, I completely missed that detail.
